packages/api/src/controllers/asset.ts

```diff
@@ -263,6 +263,13 @@ function getDownloadUrl(
   asset: WithID<Asset>,
   os: ObjectStore
 ): string {
+  // If mp4 rendition file exists, then use it as download url
+  const staticPlaybackInfos = getStaticPlaybackInfo(asset, os);
+  if (staticPlaybackInfos.length > 0) {
+    return staticPlaybackInfos[0].playbackUrl;
+  }
```

Comment:
This will change the behavior of the download URL. Instead of returning the original source file as provided by the user, which is what might be saved on IPFS for example and have a strict content integrity requirement, it will start returning an altered version of it. This could lead to unexpected content mismatches like hashes/CID not matching etc. I can see why we want to use this for recordings though, where there is no source file.

So for now, WDYT of just adding an optional boolean to this function like `useStaticRenditions` or something like that, that is only set from the recordings flow? Or perhaps only if `asset.source.type === 'recording'`, which might be safer so this also works on the asset API as well.

On the long term, I think Catalyst should start creating a static rendition with the same quality as the source and save it on the same place as we copy the source file (`/<playbackId>/video`, or probably add an extension there). This is probably more complex tho, especially since the copy is not even done on Catalyst today (still on `task-runner`)

Reply:
Ok, let's add this flag for now. It's a little hacky, but it's probably the simplest to do right now. Added here: cc9162f4b4972545974648195354f9f5895e5b87